include/zephyr/drivers/dma.h

```diff
@@ -226,6 +234,8 @@ struct dma_status {
 	uint32_t free;
 	uint32_t write_position;
 	uint32_t read_position;
+	uint32_t llp_l;
```

Comment:
The llp_l/llp_u status members are pretty specific to SOF, how it uses DMA, and Intel hardware. These should be ifdef'ed out with something like DMA_STATUS_LLP with a description of when its used.

Reply:
I have the same feeling that this is too audio specific. Can we use some more generic terms? And please define (in the commit and struct documentation) how this is different from "read_position" and "write_position".

Reply:
Something like

uint64_t copied;

perhaps? Implying the total number of something (audio samples/words/bytes, whatever is configured...) copied is probably useful, and would not require an ifdef.